### PR TITLE
Support relative paths for preferred native folder

### PR DESCRIPTION
**Pull request by gharrma**
Fixes: https://github.com/JetBrains/pty4j/issues/133

### CODE REVIEW

File: src/com/pty4j/util/PtyUtil.java [hunk 1]
```diff
@@ -24,8 +24,8 @@ public static String[] toStringArray(Map<String, String> environment) {
   private static @Nullable File getPreferredLibPtyFolder() {
     String path = System.getProperty(PREFERRED_NATIVE_FOLDER_KEY);
     File dir = path != null ? new File(path) : null;
-    if (dir != null && dir.isAbsolute() && dir.isDirectory()) {
-      return dir;
+    if (dir != null && dir.isDirectory()) {
```
Comment (segrey) [inline src/com/pty4j/util/PtyUtil.java:27]: LGTM

Maybe let's check that `System.getProperty(PREFERRED_NATIVE_FOLDER_KEY)` is non-empty?

Comment (gharrma) [reply src/com/pty4j/util/PtyUtil.java:27]: Good idea; done

**Comment by segrey** [comment]
Thanks!

**Comment by segrey** [comment]
I think a new pty4j version will be published in about two weeks.

**Comment by gharrma** [comment]
Sounds good; thanks Sergey!

**Pull request merged**